Authenticator/Source/TextFieldRowModel.swift

```diff
@@ -47,35 +47,39 @@ struct IssuerRowModel: TextFieldRowModel {
     let keyboardType: UIKeyboardType = .Default
     let returnKeyType: UIReturnKeyType = .Next
 
-    let value: String
-    let changeAction: (String) -> Form.Action
-
-    init(value: String, changeAction: (String) -> Form.Action) {
-        self.value = value
-        self.changeAction = changeAction
-    }
+    return TextFieldRowModel(
+        label: label,
+        placeholder: placeholder,
+        autocapitalizationType: autocapitalizationType,
+        autocorrectionType: autocorrectionType,
+        keyboardType: keyboardType,
+        returnKeyType: returnKeyType,
+        value: value,
+        changeAction: changeAction
+    )
 }
 
-struct NameRowModel: TextFieldRowModel {
+func NameRowModel(value value: String, returnKeyType: UIReturnKeyType, changeAction: (String) -> Form.Action) -> TextFieldRowModel<Form.Action> {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 145 characters (line_length)